assets/style/app/components/_page-footer.scss

```diff
@@ -73,20 +76,34 @@
 
     .links-social li {
       display: inline-block;
-      margin: 0 $space-base/2;
-      vertical-align: middle;
+      vertical-align: webkit-baseline-middle;
+      margin-right: 15px;
 
       svg {
         margin-top: -2px;
       }
     }
 
     .b-corp-logo {
-      svg > * {
+    svg > * {
         fill: $color-accent-gray;
       }
     }
-
+    .footer-icon-left-bottom > svg {
+      padding-bottom: 8px;
```

Comment:
can you make this padding based off of the `$space` variables that are available please